### PR TITLE
Fix variable name for mismatched genre mapping

### PR DESCRIPTION
**Pull request by paxF3E**
Line777: list `mismatched_genres` is not defined.
list `genre_labels` would be the list instead, intuitively.

### CODE REVIEW

File: outlier_mnli.ipynb [hunk 1]
```diff
@@ -774,7 +774,7 @@
     "genre_labels = np.array([labels_dict.get(x, 0) for x in selected_outlier_subset[\"genre\"]])\n",
     "for i, genre in enumerate(labels_dict.keys()):\n",
     "    x, y = x_plot[genre_labels == i], y_plot[genre_labels == i]\n",
-    "    if genre in mismatched_genres:\n",
+    "    if genre in genre_labels:\n",
```
Comment (elisno) [inline outlier_mnli.ipynb]: This subplot is a duplicate of an earlier label scatter plot to make an easier comparison with the cluster topics.

```suggestion
    "    if genre in mismatched_labels:\n",
```

In this block of code, it's intended to plot the genres in 
```python
mismatched_labels = {"nineeleven": 0, "facetoface": 1, "letters": 2, "oup": 3, "verbatim": 4}
```
with circle-markers. The `else` block handles the other genres.

**Review by elisno — CHANGES_REQUESTED** [review]
Thank for the PR @paxF3E, I really appreciate it!

It looks like some code cells got out-of-sync when making the final edits for the article.

I made a suggestion with the correct fix.

File: outlier_mnli.ipynb (hunk 1 above)
Comment (paxF3E) [reply outlier_mnli.ipynb]: Ah alright. Couldnt figure `mismatched_labels` list earlier, but perfect now.

Also, the comparison with added topic clusters is very intuitive. I was startled to witness the extended application of HDBSCAN(clustering) and UMAP(dimensionality reduction) in this context, beyond the usual use cases :D

**Comment by elisno** [comment]
@paxF3E, why did you close the PR? 😕 

You're correct that `mismatched_genres` isn't defined, but it was supposed to be `mismatched_labels`.

This still needs to be fixed, and I would gladly merge your PR if you commit the suggestion I just gave above. 🙂

**Comment by paxF3E** [comment]
Kindly check if its good now :)

**Review by elisno — APPROVED** [review]
LGTM!

**Pull request merged**